src/Evolution/Executables/Burgers/EvolveBurgers.hpp

```diff
@@ -0,0 +1,84 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <vector>
+
+#include "Domain/DomainCreators/RegisterDerivedWithCharm.cpp"
+#include "Domain/Tags.hpp"
+#include "ErrorHandling/FloatingPointExceptions.hpp"
+#include "Evolution/Actions/ComputeVolumeDuDt.hpp"  // IWYU pragma: keep
+#include "Evolution/Actions/ComputeVolumeFluxes.hpp"  // IWYU pragma: keep
+#include "Evolution/DiscontinuousGalerkin/DgElementArray.hpp"  // IWYU pragma: keep
+// for LocalLaxFriedrichsFlux
```

Comment:
What is the comment to? Normally they're on the same as the include, if not could you specify e.g. `// Equations.hpp for Local`?

Reply:
Fixed.  I'd been having trouble with IWYU complaining about having other stuff on the line, but I've found a form it is happy with now.